docs/src/pages/[platform]/connected-components/account-settings/delete-user/react.mdx

```diff
@@ -22,7 +22,7 @@ import { AccountSettings } from '@aws-amplify/ui-react';
 
 function App() {
   const handleSuccess = () => {
-    alert('password is successfully changed!')
+    alert('user has been successfully deleted')
```

Comment:
This doesn't look like a typo level change 🤔

Reply:
This one is a copy paste error 😓

Reply:
Updated description to note the changes -- would you like me to change the title too?